src_blog/_posts/New-Release-Grain-v0.4.0-Cereal.md

```diff
@@ -0,0 +1,67 @@
+---
+title: New Release! Grain v0.4.0 - Cereal
```

Comment:
Maybe we should leave it as `v0.4`, since the _real_ release is `0.4.2`?

Reply:
```suggestion
title: New Release! Grain v0.4 - Cereal
```